clusterloader2/pkg/measurement/util/phase_latency.go

```diff
@@ -102,7 +102,17 @@ func (o *ObjectTransitionTimes) CalculateTransitionsLatency(t map[string]Transit
 				klog.V(4).Infof("%s: failed to find %v time for %v", o.name, transition.To, key)
 				continue
 			}
-			lag = append(lag, latencyData{key: key, latency: toPhaseTime.Sub(fromPhaseTime)})
+			latencyTime := toPhaseTime.Sub(fromPhaseTime)
+			// latencyTime should be always larger than zero, however, in some cases, the precision of timestamp
```

Comment:
can you please shorten the lines in this comment?

Reply:
sure, could you pls take another look?